### DISCUSSION TITLE
[Feature] Share singel pictures and videos

### DISCUSSION THREAD

**Discussion by wavesswe** [immich-app/immich#3451] [Feature Request]
### The feature

Be able to share marked pictures or videos with a user or link. Not just via Album. Those pictures can then be added to an album from the user that got the picture shared to them.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by alextran1502** [comment]
You can already do this via the web interface. The UI is currently not supported yet via the mobile app
